test/board_test.rb

```diff
@@ -48,6 +48,23 @@ def test_cell_fired_at
     assert_equal true, board.cell_fired_at?("A1")
   end
 
+  def test_it_can_render_cell
```

Comment:
Added a test to make sure that the cells were rendering properly within the game.